src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -353,10 +353,6 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
           height: 64px;
         }
 
-        .weather-icon {
```

Comment:
Why was this removed?

Reply:
What if the exception icon is the main icon and not in the forecast

Reply:
Ah, I see how it can be used up there. Give me a few and I'll push a fix.

Reply:
Fixed.